src/components/SubjectInterest.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+
+    <div class="page-overview">
+      <div class="pageOne col-md-8" > I'm interested in joining mPower because I </div>
+    <br>
+    <br>
+  
+    <select class="col-md-8 offset-md-2 custom-select customizedSelect" id="select_one" v-model="selected_option_one">
+      <option disabled value="">Please select one</option>
+      <option> {{ parkinsonsMessage }}</option>
+    </select>
+  
+    <br>
+    <br>
+  
+    <div v-if="hasParkinsons" >
+      <div class="pageOne col-md-8 offset-md-4"> and I would be willing to try </div>
+      <br>
+      <br>
+      <select class="custom-select customizedSelect" id="select_two" v-model="selected_option_two">
+        <option disabled value="">Please select one</option>
+        <option>{{willing}}</option>
+      </select>
+    </div>
+  
+    <br>
+    <br>
+    <button id="next" v-on:click="clicked"> Next </button>
+  
+  </div>
+</template>
+
+<script>
+// focus on selection one on page load
+window.onload = function () {
+  this.focusElement(true, 'select_one')
+}
+
+export default {
+  name: 'subjectInterest',
+  data () {
+    return {
+      parkinsonsMessage: 'have parkinsons',
```

Comment:
I noticed some vue examples did similar things https://vuejs.org/v2/guide/#Declarative-Rendering and it also made it so that in my JS I could compare to variable names instead of strings, but I'm happy to change that.